src/main/java/com/catapi/view/CatBreedResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.catapi.view;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CatBreedResponse {
+
+    @JsonProperty("breeds")
+    private CatBreedView[] breeds;
```

Comment:
It will not deserialize to this structure. Because you don't have object with field "breeds". For your structure it's supposed to be like this:

```
{
   "breeds" :  [
       {
          "id": "abys",
           "name": "Abyssinian",
           "origin": "Egypt",
           "description": "some description",
           "wikipedia_url": "https://en.wikipedia.org/wiki/Abyssinian_(cat)"
       },
        {
           "id": "aege",
           "name": "Aegean",
           "origin": "Greece",
           "description": "some description2",
           "wikipedia_url": "https://en.wikipedia.org/wiki/Aegean_cat"
       }
   ]
}
```

but what do we really have:

```
[
       {
          "id": "abys",
           "name": "Abyssinian",
           "origin": "Egypt",
           "description": "some description",
           "wikipedia_url": "https://en.wikipedia.org/wiki/Abyssinian_(cat)"
       },
        {
           "id": "aege",
           "name": "Aegean",
           "origin": "Greece",
           "description": "some description2",
           "wikipedia_url": "https://en.wikipedia.org/wiki/Aegean_cat"
       }
   ]
```

as you can see there is no "breeds" field. So we don't need CatBreedResponse class at all